src/static_analysis/manifest.rs

```diff
@@ -65,10 +65,10 @@ pub fn manifest_analysis<S: AsRef<str>>(config: &Config,
     results.set_app_label(manifest.get_label());
     results.set_app_description(manifest.get_description());
     results.set_app_version(manifest.get_version_str());
-    results.set_app_version_num(manifest.get_version_number());
-    results.set_app_min_sdk(manifest.get_min_sdk());
+    results.set_app_version_num(manifest.get_version_number() as i32);
+    results.set_app_min_sdk(manifest.get_min_sdk() as i32);
     if manifest.get_target_sdk().is_some() {
-        results.set_app_target_sdk(manifest.get_target_sdk().unwrap());
+        results.set_app_target_sdk(manifest.get_target_sdk().unwrap() as i32);
```

Comment:
Why all this casts between `i32` and `u32`? The SDK is a u32 (in fact, in the future it will have its own enum, as it can be seen in #81). Can the version number be negative? We should be as type safe as possible, and if it's badly formatted, return an error explaining the issue.